content/concept-xrp-ledger-intro.md

```diff
@@ -0,0 +1,88 @@
+- **[Finite XRP Supply][]:** When the XRP Ledger began, 100 billion XRP were created, and no more XRP can be created after that. (Each XRP is subdivisible down to 6 decimal places, for a grand total of 100 quintillion "drops" of XRP.) The available supply of XRP decreases slowly over time as small amounts are destroyed to pay transaction costs.
+- **[Responsible Software Governance][]:** The XRP Ledger's software is maintained by a team of world-class developers at Ripple, who maintain the security of the network while adding features and stewarding future developments. Ripple acts as a steward for the software and an advocate for its interests, so the XRP Ledger ecosystem has a constructive relationship with governments and financial institutions, not an adversarial one.
+- **[Secure, Adaptable Cryptography][]:** The XRP Ledger relies on industry standard digital signature systems like ECDSA (the same scheme used by Bitcoin) but also supports modern, efficient algorithms like Ed25519. The extensible nature of the XRP Ledger's software makes it possible to add and disable algorithms as the state of the art in cryptography advances.
+- **[Smart Features for Smart Contracts][]:** The XRP Ledger has features like Escrow, Checks, and Payment Channels to support building cutting-edge financial applications, especially the [Interledger Protocol](https://interledger.org/). This toolbox of advanced features comes with separate double-checks on invariant rules, so even a glitch in the transaction implementation won't cause the XRP Ledger to become inconsistent or corrupt.
+- **[On-Ledger Decentralized Exchange][]:** In addition to all the features that make XRP useful on its own, the XRP Ledger also has a fully-functional accounting system for tracking and trading obligations denominated in any way users want, and an exchange built into the protocol. The XRP Ledger can settle long, cross-currency payment paths and exchanges of multiple currencies in atomic transactions, bridging gaps of trust with XRP.
+
+## Censorship-Resistant Transaction Processing
+[Censorship-resistant transaction processing]: #censorship-resistant-transaction-processing
+
+With (physical) coins and paper money, individuals can do business without going through a central party. Non-cryptocurrency digital assets all have a central administrator to confirm and validate transactions, who also has the power to censor or roll back transactions, or disallow some individuals from using the digital asset. (For example, if a digital money company decides a person has violated its terms of service, it can freeze or even confiscate that person's money.) XRP and other cryptocurrencies don't have a central administrator, so no one can roll back transactions, freeze XRP balances, or block someone from using them.
+
+**Note:** Non-XRP currencies issued in the XRP Ledger _can_ be frozen. For more information, see the [Freeze documentation](concept-freeze.html).
```

Comment:
Maybe add `currencies issued by *users*` to help highlight that Ripple has no special role.?